crates/ironclaw_webui_v2_static/static/vendor/fonts/fonts.css

```
@@ -0,0 +1,396 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Geist';
```

Stylelint (17.13.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)